docs/sources/developers/plugins/data-frames.md

```diff
@@ -26,45 +24,47 @@ interface DataFrame {
 }
 ```
 
-In essence, a data frame is a collection of _fields_, where each field corresponds to a column. Each field, in turn, consists of a collection of values, along with meta information, such as the data type of those values.
+### Data frame fields
+
+In essence, a data frame is a collection of _fields_, where each field corresponds to a column. Each field, in turn, consists of a collection of values, along with metadata, such as the data type of those values.
 
 ```ts
 interface Field {
```

Comment:
https://github.com/grafana/grafana/blob/c308118fc024eae0a7bf48a0b681e234e37262c3/packages/grafana-data/src/types/dataFrame.ts#L126 similarly may be out of date?

Reply:
Good catch, I've commited a fix.